Structure (#4)
Logius-standaarden/API-mod-encryption: sections/jwe.md

# Normative Design Rules

> ## Summary
> 
> Design rules can be technical rules, which should be tested automatically and functional rules which should be considered when designing and building the API.
> 
> ### List of functional rules
> 
> <ul id="functionalList"></ul>
> 
> ### List of technical rules
> 
> <ul id="technicalList"></ul>

Reviewer: ```suggestion

All design rules in this module are technical rules, which should be tested automatically.

## Summary

<ul id="technicalList"></ul>
```

Hiermee bereiden we ons voor op https://github.com/Logius-standaarden/publicatie/pull/38
